packages/mapviewer/src/modules/map/components/cesium/CesiumInteractions.vue

```diff
@@ -1,27 +1,40 @@
 <script setup>
-import { Cartesian2, Cartographic, ScreenSpaceEventType } from 'cesium'
+import {
+    Cartesian2,
+    Cartographic,
+    Color,
+    PostProcessStageLibrary,
+    ScreenSpaceEventType,
+} from 'cesium'
 import log from 'geoadmin/log'
 import { WEBMERCATOR, WGS84 } from 'geoadmin/proj'
+import { Point } from 'ol/geom'
 import proj4 from 'proj4'
-import { computed, inject, onMounted } from 'vue'
+import { computed, inject, onMounted, ref } from 'vue'
 import { useStore } from 'vuex'
 
+import LayerFeature from '@/api/features/LayerFeature.class'
 import GeoAdminGeoJsonLayer from '@/api/layers/GeoAdminGeoJsonLayer.class'
 import GPXLayer from '@/api/layers/GPXLayer.class'
 import KMLLayer from '@/api/layers/KMLLayer.class'
 import { unhighlightGroup } from '@/modules/map/components/cesium/utils/highlightUtils'
 import useDragFileOverlay from '@/modules/map/components/common/useDragFileOverlay.composable'
 import { ClickInfo, ClickType } from '@/store/modules/map.store'
+import { createPixelExtentAround } from '@/utils/extentUtils'
 import { identifyGeoJSONFeatureAt } from '@/utils/identifyOnVectorLayer'
 
 const dispatcher = { dispatcher: 'CesiumInteractions.vue' }
 
 const getViewer = inject('getViewer')
 
+const hoveredHighlightPostProcessor = ref(null)
+const clickedHighlightPostProcessor = ref(null)
```

Comment:
you might not want to store that as ref, as it will wrap them in reactivity proxies